lib/oxsdatatypes/README.md

```diff
@@ -32,6 +32,20 @@ Each datatype provides:
 * `from_be_bytes` and `to_be_bytes` methods for serialization.
 
 
+### `DateTime::now` behavior
+
+The `DateTime::now()` function needs special OS support.
+Currently:
+- For `wasm32-unknown-unknown` if the `js` feature is enabled the `Date.now()` ECMAScript API is used.
+- For all `unknown` or `none` target OSs, if the `custom-now` feature is enabled, a function computing `now` must be set in the root binary crate following:
```

Comment:
A nit, but I think `custom_ox_now` can be provided by anything (crate, external library) linked into the final DSO (whether that is a binary or library).

Personally, I also wouldn't limit the `custom-now` feature to `unknown` and `none` operating systems. If the integrator wants a custom now function, let them have it.

Reply:
(I think the most notable requirement for linkage is that the function uses the Rust ABI and types from the `oxsdatatypes` crate which in practice means it must be compiled by the same Rust compiler as that crate.)

Reply:
Indeed. I have rewritten the documentation and made `custom-now` possible on all target. Thank you!